packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/AppTests.kt

```diff
@@ -391,7 +391,7 @@ class AppTests {
             Realm.open(syncConfig).close()
 
             // Create a configuration pointing to the metadata Realm for that app
-            val lastSetSchemaVersion = 6L
+            val lastSetSchemaVersion = 7L
```

Comment:
Hmm. I don't really know the full consequence of this, but from this comment it seems that we should at least warn about not being able to downgrade 🤔

Reply:
We are opening the metadata directly instead of opening a higher schema version with an app to trigger a schema downgrade.

I am going to create a ticket to validate what is the expected behavior when downgrading an App metadata schema.

Reply:
https://github.com/realm/realm-kotlin/issues/1549

Reply:
Ahh sorry, forgot the link to the comment I was referring to: https://github.com/realm/realm-kotlin/pull/1528#discussion_r1355148801